stable/script

```diff
@@ -0,0 +1,4 @@
+SRC="$1"
+DST="$2"
+
+perl -pi -e "s:VERSION = dev:VERSION = latest:" $DST/conode/Makefile
```

Comment:
could we rename this file name to something more descriptive? e.g. `dev-to-latest`

Reply:
it's the 'standard' `stable/script` name...

Reply:
Added a comment.